pkgs/applications/radio/sdr-j-fm/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  wrapQtAppsHook,
+  pkg-config,
+  qtbase,
+  qwt,
+  fftwFloat,
+  libsamplerate,
+  portaudio,
+  libusb,
+  libsndfile,
+  airspy,
+  librtlsdr,
```

```suggestion
  rtl-sdr-librtlsdr,
```
I guess this got renamed at some point.

---

Ah, yes we have 3 different implementations for the library by now.